bin/node-template/runtime/src/lib.rs

```diff
@@ -262,7 +262,7 @@ impl pallet_transaction_payment::Config for Runtime {
 	type OperationalFeeMultiplier = ConstU8<5>;
 	type WeightToFee = IdentityFee<Balance>;
 	type LengthToFee = IdentityFee<Balance>;
-	type FeeMultiplierUpdate = ConstFeeMultiplier<FeeMultiplier>;
+	type FeeMultiplierUpdate = ConstFeeMultiplier<FeeMultiplierMin>;
```

Comment:
```suggestion
	type FeeMultiplierUpdate = ConstFeeMultiplier<FeeMultiplier>;
```
It accepts a constant, or?

Reply:
Yes, forgot to rename the `FeeMultiplierMin` to `FeeMultiplier`(min and max are the same here)